README.md

```diff
@@ -49,10 +49,10 @@ export KONG_ENV=production && export KONG_NGINX_MAIN_ENV=KONG_ENV && kong start
             "redis": {
                 "host": "localhost",
                 "port": 6379,
-				"connect_timeout": 1000
+		"connect_timeout": 1000
             }
         },
-		"docker": {
+	"docker": {
```

Comment:
Please change `docker` to something like `development` or `staging`. `docker` is confusing. @chirag-manwani

Reply:
Done on patch-1 brach. Hope they included in this PR?